app/views/home/index.html.erb

```diff
@@ -1 +1,2 @@
-<h1>Hello World!</h1>
\ No newline at end of file
+<h1>Hello World!</h1>
+<p id="notice"><%= notice %></p>
```

Comment:
Can you only get notices? What about errors, etc

Reply:
As far as I know, it's only the name of the variable? Maybe there's another way to do it or I'm going against convention.

Reply:
Oh, I might have misread this? Is this not to display flash notices? If not, is there **always** a notice present? If there isn't, then the paragraph element `<p id="notice">` should be only be conditionally displayed

Reply:
No, you were correct, I didn't know there was a specific method to display things conditionally. But AFAIK it does display only if I use a flash.

Reply:
I changed it to a success flash banner.